core/controllers/collection_editor.py

```diff
@@ -76,7 +76,8 @@ class EditableCollectionDataHandler(CollectionEditorHandler):
     """A data handler for collections which supports writing."""
 
     def _require_valid_version(self, version_from_payload, collection_version):
-        """Check that the payload version matches the given collection version.
+        """Check that the payload version matches the given collection
```

Comment:
Why has the word `version` being moved to the next line? The original sentence has only 71 characters which do not exceeds the maximum specified length of 80 characters.

Reply:
@bansalnitish

Reply:
@kevinlee12 I have addressed all these [here](https://github.com/oppia/oppia/pull/4759#issuecomment-372587884) since all required a common answer.

Reply:
sgtm!